src/components/TagPicker/index.tsx

```diff
@@ -41,22 +40,15 @@ type TagPickerProps = TagPickerOnyxProps & {
     /** Callback to submit the selected tag */
     onSubmit: () => void;
 
-    /**
-     * Safe area insets required for reflecting the portion of the view,
-     * that is not covered by navigation bars, tab bars, toolbars, and other ancestor views.
-     */
-    insets: EdgeInsets;
```

Comment:
This prop is being passed in EditRequestTagPage and IOURequestStepTag. Let's remove it

Reply:
Done